docs/Loyalty Protocol V1/Tech integration/redeem-rewards.md

```diff
@@ -113,16 +113,16 @@ Parameters:
 - `otp`: The one-time password submitted by the user.
 
 Example:
-```text
+```json
   {
-    "phone_number": "90248xx921",
+    "phone_number": "9876543219",
     "countryCode": "+91",
     "authCode": 2735,
     "otp": 596097
   }
 ```
 ### Response
-```text
+```json
   {
     "message": "successfully verified OTP & identity-associated"
```

Comment:
@Jitendra-08 Kindly re-check the response msg, now we have modified the message ig